functions/src/aqi-calculation/calculate-aqi.ts

```diff
@@ -134,10 +134,18 @@ function aqiFromPm25(pm25Concentration: number): number {
  * updates the `current-readings` AQI data and AQI-validity status. Also updates
  * the aqiBuffer in each sensor doc.
  */
+
 async function calculateAqi(): Promise<void> {
-  const sensorList = (await firestore.collection('sensors').get()).docs;
+  // Initialize the currentData map
```

Comment:
In some other files in this PR you use an actual Map object, why did we want to use the null object here? Is there a better comment then other than using the word Map?

Reply:
This is something you did way back when--this is the map that's stored in the current-readings collection in Firestore, so it's how we created a map for Firestore. Do you know of a different way to create a map for Firestore data?